<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSearchDialog</class>
 <widget class="QDialog" name="FileSearchDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>582</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Search Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Search your network for files</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="lineEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="showGrid">
        <bool>true</bool>
       </property>
       <property name="gridStyle">
        <enum>Qt::SolidLine</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="cornerButtonEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>false</bool>
       </attribute>
       <attribute name="horizontalHeaderDefaultSectionSize">
        <number>150</number>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderStretchLastSection">
        <bool>false</bool>
       </attribute>
       <row>
        <property name="text">
         <string>1</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>2</string>
        </property>
       </row>
       <row>
        <property name="text">
         <string>3</string>
        </property>
       </row>
       <column>
        <property name="text">
         <string>Item</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Owner</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Size</string>
        </property>
       </column>
       <item row="0" column="0">
        <property name="text">
         <string>gta v.iso</string>
        </property>
       </item>
       <item row="0" column="1">
        <property name="text">
         <string>aaryak</string>
        </property>
       </item>
       <item row="0" column="2">
        <property name="text">
         <string>76.1 GB</string>
        </property>
       </item>
       <item row="1" column="0">
        <property name="text">
         <string>gta v repack.iso</string>
        </property>
       </item>
       <item row="1" column="1">
        <property name="text">
         <string>harsh</string>
        </property>
       </item>
       <item row="1" column="2">
        <property name="text">
         <string>58 GB</string>
        </property>
       </item>
       <item row="2" column="0">
        <property name="text">
         <string>gta vice city.zip</string>
        </property>
       </item>
       <item row="2" column="1">
        <property name="text">
         <string>ani</string>
        </property>
       </item>
       <item row="2" column="2">
        <property name="text">
         <string>49.9 GB</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_2">
         <property name="text">
          <string>Download</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
